app/models/pieces/pawn.rb

```diff
@@ -1,2 +1,22 @@
 class Pawn < Piece
+  # rubocop:disable all
```

Comment:
It is best practice to only disable the specific cop that is failing & unable to be resolved. I assume it is ABCSize & PerceivedComplexity?